hatch test errors out

```
❯ hatch test
=============================================================================== test session starts ================================================================================
platform linux -- Python 3.12.8, pytest-8.3.4, pluggy-1.5.0
rootdir: /home/yoh/proj/repronim/reprostim
configfile: pyproject.toml
plugins: rerunfailures-14.0, mock-3.14.0, xdist-3.6.1
collected 0 items / 1 error                                                                                                                                                        

====================================================================================== ERRORS ======================================================================================
_________________________________________________________________ ERROR collecting tests/audio/test_audiocodes.py __________________________________________________________________
ImportError while importing test module '/home/yoh/proj/repronim/reprostim/tests/audio/test_audiocodes.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/audio/test_audiocodes.py:1: in <module>
    from ..audio.audiocodes import crc8
E   ImportError: attempted relative import with no known parent package
============================================================================= short test summary info ==============================================================================
ERROR tests/audio/test_audiocodes.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

if I fix for that ,then it fails due to lack of dependency... so there is more to declare in dependencies

```
❯ hatch test
=============================================================================== test session starts ================================================================================
platform linux -- Python 3.12.8, pytest-8.3.4, pluggy-1.5.0
rootdir: /home/yoh/proj/repronim/reprostim
configfile: pyproject.toml
plugins: rerunfailures-14.0, mock-3.14.0, xdist-3.6.1
collected 0 items / 1 error                                                                                                                                                        

====================================================================================== ERRORS ======================================================================================
_________________________________________________________________ ERROR collecting tests/audio/test_audiocodes.py __________________________________________________________________
ImportError while importing test module '/home/yoh/proj/repronim/reprostim/tests/audio/test_audiocodes.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/audio/test_audiocodes.py:1: in <module>
    from reprostim.audio.audiocodes import crc8
src/reprostim/audio/audiocodes.py:13: in <module>
    import sounddevice as sd
E   ModuleNotFoundError: No module named 'sounddevice'
============================================================================= short test summary info ==============================================================================
ERROR tests/audio/test_audiocodes.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================= 1 error in 0.39s =================================================================================
❯ git diff
diff --git a/tests/audio/test_audiocodes.py b/tests/audio/test_audiocodes.py
index 4a411ee..b25e09f 100644
--- a/tests/audio/test_audiocodes.py
+++ b/tests/audio/test_audiocodes.py
@@ -1,4 +1,4 @@
-from ..audio.audiocodes import crc8
+from reprostim.audio.audiocodes import crc8
 
 
 def test_crc8():
changes on filesystem:                                                                                                                                                              
 tests/audio/test_audiocodes.py | 2 +-
```

--- a/.autorc
+++ b/.autorc
@@ -1,7 +1,7 @@
 {
     "onlyPublishWithReleaseLabel": true,
     "baseBranch": "master",
-    "author": "ReproNim Bot <info@repronim.org>",
+    "author": "ReproNim Team[bot] <info@repronim.org>",
     "noVersionPrefix": true,
     "plugins": [
         "git-tag",

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -44,9 +44,6 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-test = [
-    "pytest>=8.0",
-]
 audio = [
   "sounddevice>=0.5.1",
   "scipy>=1.14.1",
@@ -59,7 +56,6 @@ psychopy = [
   "psychopy-sounddevice",
 ]
 all = [
-  "reprostim[test]",
   "reprostim[audio]",
   "reprostim[psychopy]",
 ]
@@ -94,6 +90,19 @@ include = [
 # install [all] extras
 features = [
   "all",
+]
+
+[tool.hatch.envs.hatch-test]
+features = [
+  "audio",
+  "psychopy",
+]
+extra-dependencies = [
+  "pytest>=8.0",
+  "pytest",
+  "pytest-cov",
+  "pytest-mock",
+  "pytest-xdist",
 ]
 
 [tool.hatch.envs.types]

--- a/tests/audio/test_audiocodes.py
+++ b/tests/audio/test_audiocodes.py
@@ -1,4 +1,4 @@
-from ..audio.audiocodes import crc8
+from reprostim.audio.audiocodes import crc8
 
 
 def test_crc8():

